README.md

```diff
@@ -287,6 +320,27 @@ requesterMock.EXPECT().
 	Call.Return(func(path string) string { return "result for " + path }, nil)
 ```
 
+Mock constructors
+-----------------
+
+New in [v2.11.0](https://github.com/vektra/mockery/pull/406).
+
+Mockery v2.11 introduces constructors for all mocks. This makes instantiation and mock registration a bit easier and
+less error-prone (you won't have to worry about forgetting the `AssertExpectations` method call anymore).
+
+Before v2.11:
+```go
+factory := &mocks.Factory{}
+defer factory.AssertExpectations(t)
+```
+
+You can use the type-safe expecter interface as such:
```

Comment:
This might be a terminology overload, the type-safe expecter methods are opt-in and disabled by default, so this isn't strictly true.

Reply:
LOL this is totally a mistake - I copied the section from the expecter and then modified it...so I guess I completely missed this one line :D 

And by the way, I would try to push the users to the expecter interface as much as possible, to be honest :) It's far superior way to work with mocks IMO. Just a side note... :)